Traceback (most recent call last):
  File "main.py", line 21, in <module>
    parser.parse()
  File "/home/younesbelkada/Travail/MVA/Recvis/Assignment3/recvis21_a3/utils/parser.py", line 34, in parse
    self.train_loader = torch.utils.data.DataLoader(datasets.ImageFolder(os.path.join(self.config['Dataset']['path_data'],'train_images'), transform=data_transforms), batch_size=self.config['Training']['batch_size'], shuffle=True, num_workers=1)
  File "/home/younesbelkada/Travail/venvs/recvis/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 274, in __init__
    batch_sampler = BatchSampler(sampler, batch_size, drop_last)
  File "/home/younesbelkada/Travail/venvs/recvis/lib/python3.7/site-packages/torch/utils/data/sampler.py", line 219, in __init__
    "but got batch_size={}".format(batch_size))
ValueError: batch_size should be a positive integer value, but got batch_size=32